packages/core/src/components/button/_common.scss

```diff
@@ -299,6 +299,13 @@ $button-intents: (
     background: none;
     cursor: not-allowed;
     color: $pt-text-color-disabled;
+
+    &:active,
+    &.pt-active {
```

Comment:
identical selector two blocks above, line 288, that sets the same three properties. can we merge them?